keps/487-kubectl-plugin-listing/README.md

```diff
@@ -0,0 +1,316 @@
+# KEP-487: Kubectl plugin for listing objects
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+<!--
+A table of contents is helpful for quickly jumping to sections of a KEP and for
+highlighting any additional information provided beyond the standard KEP
+template.
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+    - [Story 2](#story-2)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Version](#version)
+  - [Describe](#describe)
+  - [List](#list)
+  - [Stop](#stop)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+Provide an easy way to view and manage kueue abstractions via command line tool.
+
+## Motivation
+
+Create tools that provide useful information in an intuitive way for cluster users and could be later extended as managing tool for cluster administrators.
+As a cluster user I don’t need to dive into the difference between job and Workload concept in kueue and should be able quickly discover the job/Workload and ClusterQueue status or see the error message what is misconfigured/should be fixed.
+
+### Goals
+
+Impement basic kueue CLI commands that was described in PRD:
+
+- Display the Workload status.
+- Describe Workload.
+- List Workloads in specific queue/namespace.
+- List admitted/pending workloads
+- List all ClusterQueues.
+- Describe ClusterQueue.
+- Stop Workload.
+- Stop ClusterQueue.
+
+### Non-Goals
+
+## Proposal
+
+We propose creating a Kubectl plugin that exposes kueue specific information. The main command-line interactions will take the following shape:
+
+```bash
+$kubectl kueue <type> <command> <name> <flags> <options>
```

Comment:
I suggested current order, since it looks more usable. I think pretty common workflow would be `kubectl kueue workloads list`, and then `kubectl kueue workloads describe <name>`. So changing one last word is handy. 
However I don't have a strong opinion here. I'm fine to have the same order as `kubectl`